commandapi-platforms/commandapi-velocity/commandapi-velocity-core/src/main/java/dev/jorel/commandapi/CommandAPIVelocity.java

```diff
@@ -215,8 +215,14 @@ public void postCommandRegistration(RegisteredCommand registeredCommand, Literal
 	}
 
 	@Override
-	public LiteralCommandNode<CommandSource> registerCommandNode(LiteralArgumentBuilder<CommandSource> node) {
-		return getBrigadierDispatcher().register(node);
+	public LiteralCommandNode<CommandSource> registerCommandNode(LiteralArgumentBuilder<CommandSource> node, String namespace) {
+		LiteralCommandNode<CommandSource> builtNode = getBrigadierDispatcher().register(node);
+		if (namespace.isEmpty()) {
+			// We're done. The command already is registered
+			return builtNode;
+		}
+		getBrigadierDispatcher().getRoot().addChild(CommandAPIHandler.getInstance().namespaceNode(builtNode, namespace));
+		return builtNode;
```

Comment:
Nitpick: This is a weird way to write this code. You have two `return builtNode` statements, but you don't ever change `builtNode` after the `if` statement.

You have this:
```java
if (statement) {
  return value;
}
doStuff()
return value;
```

Which is equivalent to this:
```java
if (statement) {
  return value;
} else {
  doStuff()
  return value;
}
```

Which is equivalent to this:

```java
if (!statement) {
  doStuff()
}
return value;
```